src/app/pages/jobs/components/job-logs-sidebar/job-logs-sidebar.component.ts

```diff
@@ -9,10 +9,21 @@ import { Job } from 'app/interfaces/job.interface';
   styleUrls: ['./job-logs-sidebar.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class JobLogsSidebarComponent {
+export class JobLogsSidebarComponent implements OnChanges {
   @Input() job: Job;
+  @Input() viewType: string;
```

Comment:
Currently, it needs to search through viewType usage to understand what options are available. It would be nice to have autocomplete feature on available viewTypes. Maybe convert it to enum?